source/js/main.js

```diff
@@ -40,10 +40,25 @@ let main = {
       this.fetchHomeDataIfNeeded((data) => {
         this.injectReactComponents(data);
         this.bindGlobalHandlers();
+        this.decorateExternalLinks();
       });
     });
   },
 
+  decorateExternalLinks() {
+    Array.from( document.querySelectorAll(`a`) ).forEach((link) => {
+      let href = link.getAttribute(`href`);
+
+      // Define an external link as any URL with `//` in it
```

Comment:
Should we also check if the domain in `href` is the same as `TARGET_DOMAIN`? I'm not very familiar with how things are on Wagtail and if there's any field on Wagtail that requires admins to enter full URL instead of relative URL.

Reply:
Hmm...that's a good point. I'll work on an update for that